ElrondNetwork/elrond-go
# index only non empty genesis miniblocks

## Reasoning behind the pull request
- Fix warn for index of last tx processed, this is an edge case for genesis block on api request
  
## Proposed changes
- Index only non empty miniblocks

## Testing procedure
- Check the logs, there should be no warns like `extractExecutedTxHashes encountered invalid indices` when fetching genesis block on api. The api should be fetched soon after the testnet is started, because after several epoch, the genesis data will not be in storage anymore, so it will not be available on the api
- Check if the genesis data is indexed correctly

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
